### PR TITLE
Add `Last updated` to all LC templates

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/14009

This PR adds `Last updated` to all LC templates and generally standardizes all markup across them to support buttons, alerts, intro text, etc.

## Testing done
N/A

## Screenshots
N/A

## Acceptance criteria
- [x] Standardize all LC template markup
- [x] Add `Last updated` field to all LC templates

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/layouts/media_list_images.drupal.liquid [hunk 1]
```diff
@@ -8,88 +8,94 @@
...
           {% if fieldAlertSingle %}
-            {% include "src/site/paragraphs/alert_single.drupal.liquid" with entity = fieldAlertSingle.entity %}
+          {% include "src/site/paragraphs/alert_single.drupal.liquid" with entity = fieldAlertSingle.entity %}
           {% endif %}
 
           <!-- Buttons -->
-          {% for fieldButton in fieldButtons %}
+          <div class="vads-u-margin-y--3">
+            {% for fieldButton in fieldButtons %}
             {% include "src/site/paragraphs/button.drupal.liquid" with entity = fieldButton.entity %}
```
Comment (mpelzsherman) [inline src/site/layouts/media_list_images.drupal.liquid:35]: can we indent the include here?

File: src/site/layouts/media_list_images.drupal.liquid [hunk 2]
```diff
@@ -8,88 +8,94 @@
...
+          <!-- Repeated buttons -->
           {% if fieldButtonsRepeat %}
-            <div class="vads-u-margin-y--3">
-              {% for fieldButton in fieldButtons %}
-                {% include "src/site/paragraphs/button.drupal.liquid" with entity = fieldButton.entity %}
-              {% endfor %}
-            </div>
+          <div class="vads-u-margin-y--3">
+            {% for fieldButton in fieldButtons %}
+            {% include "src/site/paragraphs/button.drupal.liquid" with entity = fieldButton.entity %}
```
Comment (mpelzsherman) [inline src/site/layouts/media_list_images.drupal.liquid:74]: indent include?

**Review by mpelzsherman — APPROVED** [review]
LGTM! Just a few minor nits about indentation.

File: src/site/layouts/media_list_images.drupal.liquid (hunk 1 above)
Comment (kelsonic) [reply src/site/layouts/media_list_images.drupal.liquid:35]: So this is actually how our linter is set up for liquid templates, it always remove the indentation 🤔 not sure how we can modify that as I haven't dealt too much with liquid

**Review by rianfowler — APPROVED** [review]

**Pull request merged**